edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Amqp/SaslIdentityType.cs

```diff
@@ -1,12 +0,0 @@
-// Copyright (c) Microsoft. All rights reserved.
-
-namespace Microsoft.Azure.Devices.Edge.Hub.Amqp
-{
-    // TODO: What is the difference between these 2 enum values?
-
-    public enum SaslIdentityType
```

Comment:
@varunpuranik Please kindly confirm it is no longer used.  Thanks.

Reply:
I am fixing a bug related to it.. don't delete it just yet.